src/RepairManager/config/ecc-config.yaml

```diff
@@ -1,9 +1,17 @@
-cordon_dry_run: {{cnf['repair-manager']['ecc_rule']['cordon_dry_run']}}
-prometheus:
-  ip: {{cnf['repair-manager']['prometheus-ip']}}
-  port: {{cnf['repair-manager']['prometheus-port']}}
-  ecc_error_query: 'nvidiasmi_ecc_error_count{type="volatile_double"}>0'
-  step: 1m
-  interval: 10 # minutes
-  percent_threshold: 90 # percentage of data points with ecc error to be considered a bad node
-  
\ No newline at end of file
+cordon_dry_run: {{cnf['repair-manager']['ecc_rule']['cordon_dry_run']}}
+reboot_dry_run: {{cnf['repair-manager']['ecc_rule']['reboot_dry_run']}}
```

Comment:
`reboot_dry_run` doesn't seem to be used anywhere. Can we add this flag in the code to enable actual `dry_run`?